backend/regalloc/regalloc_gi_utils.ml

```diff
@@ -86,6 +84,8 @@ module Selection_heuristics = struct
     | Worst_fit
     | Random_for_testing
 
+  let default = First_available
+
   let all = [First_available; Best_fit; Worst_fit; Random_for_testing]
```

Comment:
Is there any way we can use `[@@deriving enumerate]` here?

Reply:
Unfortunately, we cannot use such
ppxes in the compiler.